stable_baselines/deepq/__init__.py

```diff
@@ -1,7 +1,7 @@
 from stable_baselines.deepq.policies import MlpPolicy, CnnPolicy, LnMlpPolicy, LnCnnPolicy
 from stable_baselines.deepq.build_graph import build_act, build_train  # noqa
 from stable_baselines.deepq.dqn import DQN
-from stable_baselines.deepq.replay_buffer import ReplayBuffer, PrioritizedReplayBuffer  # noqa
+from stable_baselines.common.replay_buffer import ReplayBuffer, PrioritizedReplayBuffer  # noqa
```

Comment:
In fact, I would call that file buffers because I would like to add a `RolloutBuffer` later (at least for v3)